snapcraft/project/_project_options.py

```diff
@@ -228,18 +223,28 @@ def __init__(
         target_deb_arch=None,
         debug=False,
         *,
-        project_dir: str = None
+        work_dir: str = None
     ) -> None:
 
-        if project_dir is None:
-            self.__project_dir = os.getcwd()
-        else:
-            self.__project_dir = project_dir
+        # Here for backwards compatibility.
+        project_dir = os.getcwd()
+        if work_dir is None:
+            work_dir = project_dir
+
+        self.use_geoip = use_geoip
+        self.parallel_builds = parallel_builds
+        self.debug = debug
```

Comment:
Are these meant to be read/write? Or should they also move back to properties?

Reply:
good catch